src/main/steps/c100-rebuild/childaddress/template.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{headingTitle}}</h1>
+<p class="govuk-body-l">{{paragraph1}}</p>
+<p class="govuk-body-m">{{paragraph2}}</p>
+
+{{ govukWarningText({
```

Comment:
@kumarmadurai Can you please use warning macro from the framework ?
{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}

Reply:
I used from framework only. please view the file from top